code/game/gamemodes/heist/heist.dm

```diff
@@ -142,42 +142,12 @@
 	return TRUE
 
 /datum/game_mode/heist/proc/forge_vox_objectives()
-
-
-
-
-	//Commented out for testing.
-	/* var/i = 1
-	var/max_objectives = pick(2,2,2,3,3)
-	var/list/objs = list()
-	while(i<= max_objectives)
-		var/list/goals = list("kidnap","loot","salvage")
-		var/goal = pick(goals)
-		var/datum/objective/heist/O
-
-		if(goal == "kidnap")
-			goals -= "kidnap"
-			O = new /datum/objective/heist/kidnap()
-		else if(goal == "loot")
-			O = new /datum/objective/heist/loot()
-		else
-			O = new /datum/objective/heist/salvage()
-		O.choose_target()
-		objs += O
-
-		i++
-
-	//-All- vox raids have these two objectives. Failing them loses the game.
-	objs += new /datum/objective/heist/inviolate_crew
-	objs += new /datum/objective/heist/inviolate_death */
-
 	if(prob(25))
 		raid_objectives += new/datum/objective/heist/kidnap
-
 	raid_objectives += new/datum/objective/steal/heist
 	//raid_objectives += new/datum/objective/steal/salvage
 	raid_objectives += new/datum/objective/heist/inviolate_crew
-	raid_objectives += new/datum/objective/heist/inviolate_death
+	//raid_objectives += new/datum/objective/heist/inviolate_death // Crew death permitted. No tears.
```

Comment:
I mean
Yeah
But also the worst thing that happens if the objective isn't properly investigated is some text is red when it should be green or vice versa
There are no actual consequences to making the wrong call